openlibrary/plugins/openlibrary/partials.py

```diff
@@ -253,6 +253,35 @@ def GET(self):
                     )
                     partial["partials"].append(str(edition_list_template))
 
+        elif component == "LazyCarousel":
+            i = web.input(
+                query="",
+                title=None,
+                sort="new",
+                key="",
+                limit=20,
+                search=False,
+                has_fulltext_only=True,
+                url=None,
+                layout="carousel",
+            )
+            i.search = i.search != "false"
+            i.has_fulltext_only = i.has_fulltext_only != "false"
+            macro = web.template.Template.globals['macros'].CacheableMacro(
```

Comment:
We are always caching the carousels that are rendered here. Should we add a way to bypass this?